### PR TITLE
Arnej/minor codec fixes 

### PR DESCRIPTION
**Pull request by arnej27959**
I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

@havardpe please review

### CODE REVIEW

File: eval/src/vespa/eval/eval/value_codec.cpp [hunk 1]
```diff
@@ -154,6 +153,9 @@ struct ContentDecoder {
     template<typename T>
     static std::unique_ptr<Value> invoke(nbostream &input, const DecodeState &state, const ValueBuilderFactory &factory) {
         std::vector<vespalib::stringref> address(state.num_mapped_dims);
+        if (state.num_blocks * state.subspace_size * sizeof(T) > input.size()) {
+            throw IllegalStateException("serialized input too big");
```
Comment (havardpe) [inline eval/src/vespa/eval/eval/value_codec.cpp:157]: or is it too small...?

**Review by havardpe — APPROVED** [review]

File: eval/src/vespa/eval/eval/value_codec.cpp (hunk 1 above)
Comment (geirst) [reply eval/src/vespa/eval/eval/value_codec.cpp:157]: Also some information about the sizes in the exception message would be nice.

**Comment by arnej27959** [comment]
@geirst @havardpe please take another look

**Review by havardpe — APPROVED** [review]

**Review by geirst — COMMENTED** [review]
👍

**Pull request merged**